es-vue-base/package.json

```diff
@@ -66,25 +68,32 @@
     "prettier-eslint": "^13.0.0",
     "rimraf": "^3.0.2",
     "rollup": "^2.52.8",
+    "rollup-plugin-peer-deps-external": "^2.2.4",
     "rollup-plugin-terser": "^7.0.2",
     "rollup-plugin-vue": "^5.1.9",
     "stylelint": "^14.9.1",
-    "vue": "^2.6.14",
+    "vue": "^2.7.8",
+    "vue-flickity": "^1.2.1",
```

Comment:
Are we beholden to the commercial licence of [flickity](https://flickity.metafizzy.co/license.html#purchasing-a-commercial-license)? Could we use CSS transitions instead? I've done this before with, happy to take a pass at that for the given component.

Reply:
Hmmn good catch on this one. I'm not sure how to interpret where we fall in that lic. Probably better to not include it and find and alternative or roll our own.